hasher-matcher-actioner/hmalib/indexers/LCCIndexer.py

```diff
@@ -0,0 +1,65 @@
+from threatexchange.signal_type.pdq_index import PDQIndex
+from hmalib.common.logging import get_logger
+from datetime import timedelta, datetime
+import time
+from threatexchange.signal_type.timebucketizer import TimeBucketizer, CSViable
+import pickle
+starttime = time.time()
+logger = get_logger(__name__)
+import typing as t
+from dataclasses import dataclass
+
+# @staticmethod
+# def get_file_contents2(
+#         since: datetime.datetime,
+        # until: datetime.datetime,
+        # type: str,
+        # storage_path: str,
+        # bucket_width: datetime.timedelta,
+        # type_class: t.Type[CSViable],
+# ):
+
+Self = t.TypeVar("Self", bound="CSViable")
+
+@dataclass(eq=True)
+class HashRecord(CSViable):
+    """
+    Example class used for testing purposes.
+    """
+    content_hash:str
+    content_id:str
+
+    def to_csv(self) -> t.List[t.Union[str, int]]:
+        return [self.content_hash, self.content_id]
+
+    @classmethod
+    def from_csv(cls: t.Type[Self], value: t.List[str]) -> Self:
+        return HashRecord(value[0],value[1])
+class LCCIndexer:
```

Comment:
nit: format. Use black on each file before submitting. @Samyakk123 can help you with it.